doc/source/whatsnew/v0.21.1.txt

```diff
@@ -22,7 +22,7 @@ Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - :meth:`Timestamp.timestamp` is now available in Python 2.7. (:issue:`17329`)
--
+- :class:`DatetimeIndex` is now timezone aware when using strings as labels  (:issue:`16785`)
```

Comment:
say for indexing

Reply:
done, but out of interest: how do you believe this can be understood in a different way?